src/handlers/HandlerFactory.ts

```diff
@@ -8,11 +9,15 @@ export default class HandlerFactory {
    *
    * @return Different kinds of transaction handlers.
    */
-  public static get(repos: Repositories): {stakeRequesteds: StakeRequestedHandler} {
+  public static get(repos: Repositories): { stakeRequesteds: StakeRequestedHandler; anchor: AnchorHandler } {
```

Comment:
I feel inconsistency in naming convention, however, it might be only me :), ignore if that's too subjective to me:

I would propose, either (looks better to me).
   StakeRequestHandler
   AnchorHandler
or
   StakeRequestedHandler
   AnchoredHandler

Reply:
Why don't we change also stakeRequesteds to stakeRequests?

Reply:
It's because of the response from the graph node. It returns `stakeRequesteds` for data. Please refer TransactionHandler Class.

Reply:
Ah, it has a long tail :). Resolving this conversation.